pkg/test/options.go

```diff
@@ -52,6 +52,7 @@ type FeatureGates struct {
 	NodeRepair              *bool
 	ReservedCapacity        *bool
 	SpotToSpotConsolidation *bool
+	AutoRelaxMinValues      *bool
```

Comment:
A feature flag like this implies to me that we are going to go through a promotion cycle for this feature, but I'm not 100% sure that we are going to go through a promotion cycle to promote this to be the default behavior at all times. To me, this is just a setting that gives you a change of behavior so it would better if it was its own environment variable or even changing the name to something like `MinValuesPolicy` or `WhenMinValuesUnsatsfiable` might give you more flexibility in the future to extend the enum

Reply:
@jonathan-innis Good point, I'll update the feature flag.

@jmdeal deliberately want to start off with a feature flag. As we get more feedback around this feature, we can decide to graduate this to the NodePool API.